examples/amm/src/main.rs

```diff
@@ -20,17 +21,17 @@ struct Miner {
     pub compiled_swap_nu: CompiledFheProgram,
 
     /// The Miner's runtime
-    runtime: Runtime,
+    runtime: FheRuntime,
 }
 
 impl Miner {
     pub fn setup() -> Result<Miner, Error> {
         let app = Compiler::new().fhe_program(swap_nu).compile()?;
 
-        let runtime = Runtime::new(app.params())?;
+        let runtime = GenericRuntime::new_fhe(app.params())?;
```

Comment:
All of these examples import `GenericRuntime`, but the docs in that module all reference the type synonym `Runtime`. I think the examples and docs should be in agreement on the intended use. (Personally I'd keep it as `Runtime` throughout.)

Reply:
This was a bad F2 refactor. I've fixed this.